source/libtomo/misc/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+
+# Locate sources and headers for this project headers are included for IDEs
+
+file(GLOB tomo_headers
+    ${CMAKE_CURRENT_LIST_DIR}/../../include/*.h
+    ${CMAKE_CURRENT_LIST_DIR}/../../include/*.hh
+    ${CMAKE_CURRENT_LIST_DIR}/../../include/*.hpp
+    ${CMAKE_CURRENT_LIST_DIR}/../../include/*.icc)
```

Comment:
Is this variable inherited from the top level CMakeList? If so, we shouldn't define it again.